src/models/workouts.js

```diff
@@ -13,22 +13,27 @@ const schema = new Schema({
   },
   exercises: [
     {
-      name: String,
-      description: String,
+      type: {
+        name: String,
+        description: String,
+      },
+      reps: {
+        type: Number,
+        required: true,
+      },
+      sets: {
+        type: Number,
+        required: true,
+      },
     },
   ],
-  reps: {
-    type: Number,
-    required: true,
-  },
-  sets: {
-    type: Number,
-    required: true,
-  },
   userId: {
     type: String,
     required: true,
   },
+
 });
-const Workout = mongoose.model('workouts', schema);
+
+const Workout = mongoose.model('Workout', schema);
```

Comment:
I'd change the name of this to MuscleGroups as that reflects the data that it actually is.

We probably need to decide where Exercises have an id generated by us, or they exist as a separate model.

e.g. MuscleGroups has an array of ExerciseIds which are of type ObjectId, and can be populated by mongoose.
You'd then have an Exercise model where you would put all the exercises in.

That way when a user saves their created 'Workout' it will have savedExercises as a property for example, that is an array of ids from the exercises that mongoose can again populate.

The data here is quire referential.